pkg/summarizer/summary.go

```diff
@@ -97,8 +97,8 @@ func Update(ctx context.Context, client gcs.Client, configPath gcs.Path, concurr
 					errCh <- errors.New(dash.Name)
 					continue
 				}
-				log.WithField("summary", sum).Info("summarized")
 				if !confirm {
+					log.WithField("summary", sum).Info("Summarized")
```

Comment:
Should we still log _that_ we summarized the dashboard in either case, and the full summary only when !confirm?

Reply:
Done